src/python/WMCore/MicroService/Unified/MSTransferor.py

```diff
@@ -415,45 +391,143 @@ def makeTransferRequest(self, wflow):
         self.rseQuotas.evaluateQuotaExceeded()
         return success, response
 
-    def _subscribeData(self, subscriptionObj, wflowName, dsetName):
+    def makeTransferRucio(self, wflow, dataIn, subLevel, blocks, dataSize, nodes, nodeIdx):
```

Comment:
@todor-ivanov FYI Todor, this is more or less what you will need for the output data placement

Reply:
Thanks @amaltaro  I am looking at your code now.